plugins/flytekit-papermill/flytekitplugins/papermill/task.py

```diff
@@ -277,16 +270,14 @@ def execute(self, **kwargs) -> Any:
         output_list = []
 
         for k, type_v in self.python_interface.outputs.items():
-            if k == self._IMPLICIT_OP_NOTEBOOK:
-                output_list.append(self.output_notebook_path)
```

Comment:
Are you just removing these?

Reply:
cc @rahul-theorem I know you're using papermill plugin, do you use these paths in the output?